life_expectancy/data_load_save.py

```diff
@@ -0,0 +1,59 @@
+from inspect import getsourcefile
+from os.path import dirname, abspath, join
+import pandas as pd
+
+
+def __get_current_directory_full_path() -> str:
+    """
+    Returns the absolute path of the directory containing the current
+    Python file.
+    :return: A string representing the absolute path of the directory
+    containing the current Python file.
+    """
+    current_file_path: str = abspath(getsourcefile(lambda: 0))
+    current_dir_path: str = dirname(current_file_path)
+
+    return current_dir_path
+
+
+def load_data(
+        path: str = None
+) -> pd.DataFrame:
+    """
+    Loads the 'eu_life_expectancy_raw.tsv' data file from the 'data' folder
+    :param path: path to file to load.
+    :return: Pandas dataframe with data
+    :raises FileNotFoundError: If the 'eu_life_expectancy_raw.tsv' file cannot
+     be found.
+    """
+    # Ensure path
+    if path is None:
+        path = join(
+            __get_current_directory_full_path(),
+            "data",
+            "eu_life_expectancy_raw.tsv"
+        )
+    # Load data
+    raw_data: pd.DataFrame = pd.read_csv(
```

Comment:
you say that FileNotFoundError is going to be raised, but you don't raise any exceptions inside your code. Are you expecting to be raised by pandas? if that's the case, if you need to update the library for some reason, you may have to change this. Why don't you raise it by yourself and with a custom Exception message?

Reply:
Yes, the expectation is that this error is raised by Pandas. I did not create a custom Exception handler as this is not relevant for the function itself - I do not have a special way of dealing with this error. The `:raises ` tag should give some hints on possible exceptions being raised inside the code